src/args.c

```diff
@@ -798,9 +799,10 @@ void parse_args(int argc, char* argv[])
 			printf("\t                    interfaces\n");
 			printf("\t                    Append %s-x%s to force scan on all\n", cyan, normal);
 			printf("\t                    interfaces and scan 10x more often\n");
-			printf("\t%s--totp%s              Generate valid TOTP token for 2FA\n", green, normal);
+			printf("\t%s--totp%s          Generate valid TOTP token for 2FA\n", green, normal);
 			printf("\t                    authentication (if enabled)\n");
-			printf("\t%s-h%s, %shelp%s            Display this help and exit\n\n", green, normal, green, normal);
+			printf("\t%s--%s (options)	  Pass options to internal dnsmasq resolver\n", green, normal);
```

Comment:
![Screenshot from 2023-06-10 13-11-08](https://github.com/pi-hole/FTL/assets/16748619/007f3198-e74a-4cca-b729-bc4241b70daa)

This kind of sticks out in a not so nice way

Reply:
![Screenshot at 2023-08-01 22-17-44](https://github.com/pi-hole/FTL/assets/26622301/cd2f9568-29ed-438e-b245-a82ba10fcefa)